.circleci/config.yml

```diff
@@ -323,7 +323,7 @@ jobs:
       - checkout
       - <<: *restore_dep
       - <<: *restore_lib
-      - run: rm -rf app-wallet-interface channel-provider channel-client \
+      - run: cd packages && rm -rf app-wallet-interface channel-provider channel-client \
```

Comment:
😱

Reply:
All credit goes to @spersico for catching this one.